config/settings.yml

```diff
@@ -13,3 +13,9 @@ plugins:
   retort_disabled_categories:
     default: ""
     client: true
+  retort_max_per_day:
+    default: 50
+    client: true
```

Comment:
Hm, there's no clientside code in this PR, so I'd be surprised if this needs to be `client: true`, unless you're thinking you need it for a future PR?

Reply:
Yeah fair. I've removed it.

https://github.com/gdpelican/retort/pull/38/commits/b55e3a4ea2f412ffbf0bc75d67ec4caad0a2b130